tests/smoke_tests/run_smoke_test.py

```diff
@@ -0,0 +1,182 @@
+import asyncio
+import logging
+from pathlib import Path
+
+import torch
+import yaml
+from flwr.common.typing import Config
+from six.moves import urllib
+
+from examples.fedprox_example.client import MnistFedProxClient
+from fl4health.utils.metrics import Accuracy
+
+logging.basicConfig(format="%(asctime)s %(levelname)-8s %(message)s", level=logging.INFO, datefmt="%Y-%m-%d %H:%M:%S")
+logger = logging.getLogger()
+
+
+async def run_smoke_test(
+    config_path: str = "tests/smoke_tests/config.yaml",
+    dataset_path: str = "examples/datasets/mnist_data/",
+) -> None:
+    with open(config_path, "r") as file:
+        config = yaml.safe_load(file)
+
+    _preload_dataset(dataset_path, config)
+
+    # Start the server and capture its process object
+    logger.info("Starting server...")
+    server_process = await asyncio.create_subprocess_exec(
+        "nohup",
```

Comment:
Dumb question, but I'm not familiar with the asyncio library. If we're performing an async execution with the library, do we need the nohup part of this call? I normally use nohup to "force" an async execution on the terminal but it feels like maybe the asyncio library would already do that?